AnkiDroid/src/main/java/com/ichi2/preferences/SliderPreference.kt

```diff
@@ -62,6 +64,20 @@ class SliderPreference(context: Context, attrs: AttributeSet? = null) : Preferen
     private var displayValue: Boolean = false
     private var displayFormat: String? = null
 
+    // flyweight pattern: all listeners for an instance of the class as the same
+    // We also need to avoid any method-level closures: this callback is unused
+    // the second time `onBindViewHolder` is called
+    private val onTouchListener = object : Slider.OnSliderTouchListener {
+        override fun onStartTrackingTouch(slider: Slider) {}
+
+        override fun onStopTrackingTouch(slider: Slider) {
+            val sliderValue = slider.value.toInt()
+            if (callChangeListener(sliderValue)) {
```

Comment:
```suggestion
            if (sliderValue != value && callChangeListener(sliderValue)) {
```

different from the original code, but that stop sending unchanged results